packages/devextreme-scss/scss/widgets/generic/chat/_sizes.scss

```
@@ -0,0 +1,43 @@
+@use "../sizes" as *;
+
+// adduse
+
+$chat-bubble-border-radius: null !default;
+$chat-bubble-padding: null !default;
+$chat-bubble-container-gap: 4px !default;
+$chat-messagebox-padding: null !default;
+$chat-messagebox-textarea-min-height: null !default;
+$chat-messagebox-textarea-border-radius: null !default;
```

```suggestion
  $chat-messagebox-textarea-border-radius: $texteditor-input-border-radius !default;
```

---

We decided that the Material and Generic themes don't have technical issues to set custom border-radius. That's why they are different from the default ones.

---

Yes, but you unify styles between themes and you need set the same value of border-radius is used. Let's declare it the following way:
```suggestion
$chat-messagebox-textarea-border-radius: $texteditor-input-border-radius !default;
```

---

Don't forget remove redundant duplications ($chat-messagebox-textarea-border-radius: 4px !default;) inside if blocks. Use the same approach for fluent.